sentry/src/main/java/io/sentry/TracePropagationTargets.java

```diff
@@ -15,9 +15,16 @@ public static boolean contain(final @NotNull List<String> origins, final @NotNul
       return false;
     }
     for (final String origin : origins) {
-      if (url.contains(origin) || url.matches(origin)) {
+      if (url.contains(origin)) {
         return true;
       }
+      try {
+        if (url.matches(origin)) {
+          return true;
+        }
```

Comment:
```suggestion
    return url.matches(origin)
```
could be just this?

Reply:
Oops, no we can't because then we'd just return whether the first entry in the list matches but we want to check all of them and only return `false` if none of them match.